core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.retry;
+
+import static com.linecorp.armeria.client.retry.FixedBackoff.NO_DELAY;
+
+/**
+ * Control back off between attemtps in a single retry operation.
+ */
+@FunctionalInterface
+public interface Backoff {
+    /**
+     * Returns a {@link Backoff} that provides zero interval.
```

Comment:
```that will never wait between attempts.```

I think phrasing the comments in terms of waiting instead of interval is easier to read.